index.html

```diff
@@ -138,9 +139,7 @@ <h2 id="pj1-h2-alt">Facebook 360</h2>
                 </li>
               </ul>
          </div>   
-         <div class="resume">
-            <a href="#">See Project</a> 
-         </div>    
+         <button class="resume" id="project-one-details" type="button">See Project</button>   
        </section>
```

Comment:
All the cards should be generated dynamically through javascript. None of the cards should be strongly typed directly in `index.html` file. Consider fixing it.

Reply:
All the code is generated using javascript and style using css. The above screenshot is just an id for the existent button which is supposed to generate the pop-up.